demo/psa/prov-verif-e2e/README.md

```diff
@@ -0,0 +1,154 @@
+# PSA Demonstration Steps
+
+This document explains all the steps one need to follow to successfully run
+end to end PSA Demonstration
+
+## Preconditions
+
+* One need to install `jq` and `curl`.
+
+* Commands below assume execution in a Bourne-shell. Please adjust appropriately in case any other shell is used.
```

Comment:
```suggestion
* Commands below assume execution in a Bourne-compatible shell. Please adjust appropriately in case any other shell is used.
```

Note: no one  actually uses Bourne shell any more*. People mostly use bash (Bourne Again SHell) or zsh, both of which a re Bourne-compatible (as some other shells are).

(* well, apart from running bash in Bourne mode via '/bin/sh' alias)